### ISSUE TITLE
Clearpay payment method icon's aria-label is "Afterpay" for UK sites

### ISSUE THREAD

**Issue by csmcneill** [Automattic/woocommerce-payments#8137]
### Describe the bug

Related to #8063 and #8046: after 7.2.0, UK merchants will now offer Clearpay instead of Afterpay.

However, if a UK merchant accepts a Clearpay purchase, the icon that loads in **Payments > Transactions** has an `aria-label="Afterpay"` attribute. It should be `aria-label="Clearpay"`.

### To Reproduce

1. Install and activate 7.2.0+ (including a test build like [7.2.0-test-2](https://github.com/Automattic/woocommerce-payments/releases/tag/7.2.0-test-2))
2. Create a UK site.
3. Enable Clearpay (you may need to add a support email/phone first).
4. Purchase a product between £1.00–£1,200.00 GBP via Clearpay.
5. Navigate to **Payments > Transactions**.
6. Hover and/or inspect the payment method icon.

### Actual behavior
The `aria-label` attribute is `Afterpay` on a UK site, when the regional name for this payment method is `Clearpay` in the UK.

### Screenshots

![afterpay clearpay aria label](https://github.com/Automattic/woocommerce-payments/assets/82999806/b4e6452f-d7a2-48f7-a246-b90334931f76)

**Comment by zmaglica** [comment]
This is the type of issue that team Pulsar or team Heisenberg can work on. @kalessil and @FangedParakeet could you please decide which team could focus on this issue? 

For now, I will move it to the Other Teams queue, as a part of the Gamma triage process.

**Comment by kalessil** [comment]
@FangedParakeet: given Heisenberg started to work on Affirm branding in https://github.com/Automattic/woocommerce-payments/pull/8046, I think ya folks are the better pick here. WDYTH?

**Comment by FangedParakeet** [comment]
@kalessil, yep, no worries: Heisenberg will pick this up, despite it being relatively low priority for the time being. 👍
